content/master/concepts/pods.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Crossplane Pods
+weight: 1
+description: Background on the components installed with Crossplane and their functions.
+---
+
+The base Crossplane installation consists of two Pods, the `crossplane` pod and
+the `rbac-manager` pod. Both pods install in the `crossplane-system`
+namespace by default. 
+
+
+## Crossplane pod
+
+### Init pod
+Before starting the core Crossplane pod an _init_ Pod runs. The init pod is
+responsible for setting parameters related to the core Crossplane pod and
+installing the core Crossplane 
+[Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
+(`CRDs`).
+
+The settings the init pod sets include installing Provider or Configuration 
+packages with Crossplane, customizing the namespace Crossplane installs in and 
+defining webhook configurations. 
+
+The core CRDs installed by the init pod include: 
+* Composite Resource Definitions, Compositions, Configurations and Providers
+* `locks` to manage package dependencies
+* `controllerconfigs` to apply settings to installed Providers
+* `storeconfigs` for connecting external secret stores like 
+[Hashicorp Vault](https://www.vaultproject.io/)
+
+{{< hint "note" >}}
+
+The [Install Crossplane]({{< ref "../software/install" >}}) section has more
+information about customizing the Crossplane install.
+{{< /hint >}}
+
+The init pod completes and starts the Crossplane core pod. 
+
+### Core pod
+
+The main Crossplane pod, called the _core_ pod, enforces
+the desired state of Crossplane resources, manages leader elections and process
+webhooks. 
+
+#### Reconcile loop
+
+The core pod operates on a _reconcile loop_, constantly checking the status of
+deployed resources and correcting any "drift." After checking a resource
+Crossplane waits some time and checks again.
+
+{{<hint "note" >}}
+The Crossplane pod only reconciles core Crossplane components, including Claims
+and Composite Resources. Providers are responsible for reconciling their managed
+resources. 
+{{< /hint >}}
+
+Crossplane monitors resources in one of two ways:
+1. A Kubernetes _[watch](https://kubernetes.io/docs/reference/using-api/api-concepts/#efficient-detection-of-changes)_
+2. Periodic polling
+
+Crossplane requests that Kubernetes notifies Crossplane of any changes on
+objects. This notification tool is a _watch_. 
+
+Watched objects include Providers and Composite Resource Definitions.
+
+For objects that Kubernetes can't provide a watch for, Crossplane
+periodically poll the resource to find it's state. The default polling rate is
+one minute. Change the polling rate with the `---poll-interval` pod argument.
```

Comment:
Where should one change the polling rate? Can we be more specific than just stating to add it as a pod argument. Also is there any considerations that need to be taken into account when changing the poll-interval?

Reply:
The polling interval is an input to the helm chart for Crossplane deployment

Reply:
I added some text on the impact of changing the poll-interval. 

@jeanduplessis are you okay with the note on "Read the Change Pod Settings section...." for information on applying the change or do you want more here?